package.json

```diff
@@ -17,38 +17,37 @@
   ],
   "repository": "eclipse-velocitas/cli",
   "dependencies": {
-    "@oclif/core": "^2.9.0",
```

Comment:
Do you think about having strict version defined? It will fix issue  with regenerated "NOTICE..." file even if trivial change is going to be committed.

Reply:
can be discussed, i personally dont think its necessary. even not with this pr